src/main/resources/assets/illuminations/lang/en_us.json

```diff
@@ -1,10 +0,0 @@
-{
-  "item.illuminations.bug_net": "Bug Net",
-  "item.illuminations.firefly": "Firefly",
-  "block.illuminations.firefly_in_a_bottle": "Firefly in a Bottle",
-  "block.illuminations.firefly_grass": "Firefly Grass",
-  "block.illuminations.firefly_tall_grass": "Firefly Tall Grass",
-  "entity.illuminations.firefly": "Firefly",
-  "entity.illuminations.lightning_bug": "Lightning Bug",
-  "entity.illuminations.will_o_wisp": "Will O'Wisp"
-}
```

Comment:
Why did you delete the English localization file ? Could you please revert that change ?

Reply:
Oh sorry
I'll revert it